File: 01-reportRepair/main.py
input_string = open("input.txt").read().strip().split("\n")

values = [int(x) for x in inputString]

for i in values:
    for v in values:
        if v + i == 2020:
            print("Part one answer is:", v * i)

print()

for i in values:
    for v in values:
        for x in values:
            if v + i + x == 2020:
                print("Part two answer is:", v * i * x)

File: 01-reportRepair/main.go
package main

import (
	"fmt"
	"io/ioutil"
	"strconv"
	"strings"
)

func main() {
	inputBytes, _ := ioutil.ReadFile("input.txt")
	inputSlice := strings.Split(strings.TrimSpace(string(inputBytes)), "\n")

	var values []int
	for _, v := range inputSlice {
		str, _ := strconv.Atoi(v)
		values = append(values, str)
	}

	for _, i := range values {
		for _, v := range values {
			if v+i == 2020 {
				fmt.Println("Part one answer is:", v*i)
			}
		}
	}

	fmt.Println()

	for _, i := range values {
		for _, v := range values {
			for _, x := range values {
				if v+i+x == 2020 {
					fmt.Println("Part two answer is:", v*i*x)
				}
			}
		}
	}

}
